src/iovolpiano.cpp

```diff
@@ -0,0 +1,133 @@
+{
+    m_doc->Reset();
+    m_doc->SetType(Raw);
+    // The mDiv
+    Mdiv *mdiv = new Mdiv();
+    mdiv->m_visibility = Visible;
+    m_doc->AddChild(mdiv);
+    // The score
+    Score *score = new Score();
+    mdiv->AddChild(score);
+    // the section
+    Section *section = new Section();
+    score->AddChild(section);
+
+    Staff *staff = new Staff(1);
+    Measure *measure = new Measure(UNMEASURED, 1);
+    Layer *layer = new Layer();
+    layer->SetN(1);
+
+    staff->AddChild(layer);
+    measure->AddChild(staff);
+    section->AddChild(measure);
+
+    data_ACCIDENTAL_WRITTEN accidVal = ACCIDENTAL_WRITTEN_NONE;
+
+    static std::map<char, std::pair<data_PITCHNAME, int>> notes = { { '9', { PITCHNAME_g, 3 } },
+        { 'a', { PITCHNAME_a, 3 } }, { 'b', { PITCHNAME_b, 3 } }, { 'c', { PITCHNAME_c, 4 } },
+        { 'd', { PITCHNAME_d, 4 } }, { 'e', { PITCHNAME_e, 4 } }, { 'f', { PITCHNAME_f, 4 } },
+        { 'g', { PITCHNAME_g, 4 } }, { 'h', { PITCHNAME_a, 4 } }, { 'j', { PITCHNAME_b, 4 } },
+        { 'k', { PITCHNAME_c, 5 } }, { 'l', { PITCHNAME_d, 5 } }, { 'm', { PITCHNAME_e, 5 } },
+        { 'n', { PITCHNAME_f, 5 } }, { 'o', { PITCHNAME_g, 5 } }, { 'p', { PITCHNAME_a, 5 } },
+        { 'q', { PITCHNAME_b, 5 } }, { 'r', { PITCHNAME_c, 6 } }, { 's', { PITCHNAME_d, 6 } } };
+
+    static std::map<char, char> liquescents = { { ')', '9' }, { 'A', 'a' }, { 'B', 'b' }, { 'C', 'c' }, { 'D', 'd' },
+        { 'E', 'e' }, { 'F', 'f' }, { 'G', 'g' }, { 'H', 'h' }, { 'J', 'j' }, { 'K', 'k' }, { 'L', 'l' }, { 'M', 'm' },
+        { 'N', 'n' }, { 'O', 'o' }, { 'P', 'p' }, { 'Q', 'q' }, { 'R', 'r' }, { 'S', 's' } };
+
+    for (char ch : volpiano) {
+        if (notes.contains(ch) || liquescents.contains(ch)) {
+            const bool liquescent = liquescents.contains(ch);
+            if (liquescent) {
+                ch = liquescents.at(ch);
+            }
+            Note *note = new Note();
+            auto [pname, oct] = notes.at(ch);
+            note->SetPname(pname);
+            note->SetOct(oct);
+            if (accidVal != ACCIDENTAL_WRITTEN_NONE) {
+                Accid *accid = new Accid();
+                accid->SetAccid(accidVal);
+                accid->IsAttribute(true);
+                note->AddChild(accid);
+                accidVal = ACCIDENTAL_WRITTEN_NONE;
+            }
+            if (liquescent) {
+                note->SetCue(BOOLEAN_true);
+            }
+            layer->AddChild(note);
+        }
+        else if (ch == 'y' || ch == 'i' || ch == 'z') {
+            accidVal = ACCIDENTAL_WRITTEN_f;
+        }
+        else if (ch == 'Y' || ch == 'I' || ch == 'Z') {
+            accidVal = ACCIDENTAL_WRITTEN_n;
+        }
+    }
+
+    // add minimal scoreDef
+    StaffGrp *staffGrp = new StaffGrp();
+    StaffDef *staffDef = new StaffDef();
+    staffDef->SetN(1);
+    staffDef->SetLines(5);
+    // Arbitray clef
+    Clef *clef = new Clef();
+    clef->IsAttribute(true);
```

Comment:
Even if we don't care for it for now, the clef _is_ encoded, so I'd prefer to have it as element:
```suggestion
    clef->IsAttribute(false);
```

Reply:
I think an attribute is fine for now.